README.md

```diff
@@ -1,3 +1,5 @@
+**This repo is not connected with ringwormGO**
+
```

Comment:
This probably not meant to be a part of the PR :)

Reply:
![image](https://user-images.githubusercontent.com/83548580/139855199-89013819-4746-4c94-9dbd-1acd59446f4f.png)

Reply:
It's not possible to ignore, because on merge this change will go into master branch.

Reply:
I fixed. 
![image](https://user-images.githubusercontent.com/83548580/139859521-f354a368-a848-401b-be86-ba642ba5cf9c.png)